docs/articles/documentation/why-testcafe/README.md

```diff
@@ -0,0 +1,126 @@
+
+Emulate HTTP responses to feed sample data to your app, troubleshoot connectivity errors, and cheat downtime.
+
+### Client-side injections
+
+Execute custom client-side code to parse the web page, examine its state, or even add extra dependencies.
+
+### Frames and windows
+
+Make use of multiple browser windows to test out complex user interactions. Switch to and from iframes with ease and stability.
+
+### Concurrent tests
+
+Run your tests in multiple browsers at once to quickly uncover browser-specific errors.
+
+## Made with automation in mind
+
+### Automated waiting
+
+Manual timeouts are a thing of the past. Asynchronous from the ground up, TestCafe knows when to wait and what to wait for before a test can continue.
+
+### Automated authentication scripts
+
+Save and activate user roles to easily switch between different user accounts.
+
+### Hooks
+
+Keep your code clean with hooks — routines that are repeated before and after each test.
+
+## Helps you help yourself
+
+### TestCafe Studio
+
+The GUI and the test recorder built into TestCafe Studio make our product accessible to people with zero programming experience.
+
+### TestCafe Live
+
+Enable the built-in [Live Mode](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/run-tests.html#live-mode) to reload your test every time you make changes to the test file.
+
+### Debugging assistance
+
+Pause your tests to examine the app and troubleshoot errors.
+
+## Community-driven
+
+### Free and Open Source
+
+TestCafe is distributed for free under the MIT license.
+
+### Actively maintained
+
+The TestCafe team is quick to respond to GitHub issues and StackOverflow questions. Community suggestions shape the future of the product.
+
+### In-house support
+
+All TestCafe Studio Pro customers (subscribers and trial users alike) receive prompt, proactive support directly from the package maintainers.
+
+
+## How Does TestCafe Work?
+
+To simulate user activity, the automation engine behind TestCafe takes over browsers and the web pages they display.
+
+### Client-Server Architecture
+
+TestCafe's hybrid client-server architecture lets it execute both system-level and in-browser code.
+
+* TestCafe uses [high-level system APIs](https://github.com/DevExpress/testcafe-browser-tools) to launch and manage browsers. This is necessary to control the test execution process.
+* TestCafe tests are Node.js scripts. They can launch services and applications, read and write system files, make use of your favorite libraries.
+* TestCafe uses client-side automation scripts to execute in-browser actions. This is how our testing library handles asynchronous events, simulates user activity, and executes user-defined JavaScript.
+
+Early versions of TestCafe ran entirely in the browser. Moving to a hybrid architecture allowed us to improve the tests' stability and expand the framework's range of testing capabilities. 
+
+### Page Proxying: Bridging the Client-Server Gap
```

Comment:
I get that people use the phrase "bridging the gap" a lot... But I guess that "gap" is something that's missing - it's an issue we're fixing (bridging). In this case you seem to call the Client-Server architecture "a gap", but that's something we decided to do. So it seems like we're simply solving the problems we created ourselves. In other words, I wouldn't use that phrase in this context. 

Maybe we simply this to the first part only?

```suggestion
### Page Proxying
```